.github/workflows/acceptance-tests-runner.yml

```
@@ -233,9 +233,10 @@ jobs:
       with:
         filters: |
           advanced_cluster:
-            - 'internal/service/advancedcluster/*.go'
+            - 'internal/service/advancedcluster/!(*_test).go'
```

```suggestion
            - 'internal/service/advancedcluster/!(*_test).go'  # matches any file change except test files
```

nit, feeel free to discard

---

done in 053deb331cb7c3189a7de3ffa47e597ba5721ac1